modals-notifications/es5/src/js/modal.js

```diff
@@ -0,0 +1,60 @@
+function Modal(config) {
+	Parent.prototype.constructor.apply(this, arguments);
+
+	this.html = '' +
+		'<div class="overlay overlayJS"></div>' +
+		'<div class="modal modalWrapJS">' +
+			'<div class="modal__wrap">' +
+				'<div class="modal__main">' +
+					'<img class="modal__icon" src="img/' + this.config.icon + '.svg" alt="modal-icon">' +
+					'<span class="modal__title">' + this.config.title + '</span>' +
+					'<p class="modal__desc">' + this.config.desc + '</p>'  +
+				'</div>' +
+			'<span class="modal__close btn-style btn-style_gray modalCloseJS">Закрыть</span>' +
+			'</div>' +
+		'</div>'
+}
+
+extend (Modal, Parent);
+
+Modal.prototype = {
+	show: function () {
+		var parentWarp = document.querySelector('.modalJs');
+
+		if(parentWarp) {
+			this.parent = parentWarp;
+
+		} else {
+			this.parent.setAttribute('class', 'modal-container modalJs');
+
+			document.body.insertAdjacentElement('beforeend', this.parent);
```

Comment:
show method should not insert HTML in the code, nor hide should remove content. 
Show should show (display: block)
Hide should hide (display: none)

All the constructions and HTML creations should be put into Constructor